server/neptune/gateway/event/comment_handler.go

```diff
@@ -77,40 +70,19 @@ func (p *CommentEventWorkerProxy) Handle(ctx context.Context, request *http.Buff
 		return p.forwardToSns(ctx, request)
 	}
 
-	if shouldAllocate {
-		if cmd.ForceApply {
-			p.logger.InfoContext(ctx, "running force apply command")
-			if err := p.vcsClient.CreateComment(event.BaseRepo, event.PullNum, warningMessage, ""); err != nil {
-				p.logger.ErrorContext(ctx, err.Error())
-			}
-			return p.scheduler.Schedule(ctx, func(ctx context.Context) error {
-				return p.forceApply(ctx, event)
-			})
-		}
-
-		// notify user that apply command is deprecated on platform mode
-		if cmd.Name == command.Apply {
-			p.handleLegacyApplyCommand(ctx, event, cmd)
+	if shouldAllocate && cmd.ForceApply {
```

Comment:
we still need to build root config here to determine if platform mode is enabled for FA right? I can address this in my PR if you want